README.md

```diff
@@ -1,2 +1,20 @@
-# mocha-working-examples
-Repository of working examples of common configurations using mocha.
+# Mocha working examples
+A set of working examples to help demontrate common configurations using Mocha.
```

Comment:
```
A set of examples to help demonstrate common configurations using Mocha.
```

Reply:
Isn't the fact they are _working_ an important part of this repo? I tend to think of examples as static and not fully flushed out to the degree these are. The potential to be scaffolding for a production product.

Reply:
> Isn't the fact they are working an important part of this repo?

Important isn't quite the right word... in my mind, _expected_ would be.
We're providing the examples, so I expect them to be better than the average SO reply.

Reply:
Perhaps your right. But I don't know if most people will make that distinction. I just thought it was that bit more expressive and explanatory this way. Tbh I still do.